### DISCUSSION TITLE
Lesson7 More Cheatcodes - VSCode displays a red squiggly warning line, but the tests still pass for testFundUpdatesFundedDataStructure()

### DISCUSSION THREAD

**Discussion by lilsherryG** [Cyfrin/foundry-full-course-cu#1485] [Q&A] [answered]
# VSCode displays a red squiggly warning line, but the tests still pass.

## in FundMeTest.t.sol 
```solidity

    function testFundUpdatesFundedDataStructure() public {
        fundMe.fund{value: 10e18}();

        uint256 amountFunded = fundMe.getAddressToAmountFunded(address(this));
        assertEq(amountFunded, 10e18);
    }
```

## The warning is 

> Member "getAddressToAmountFunded" not found or not visible after argument-dependent lookup in contract FundMe.solidity(9582)

Though I ignored this and passed the test(which means this member really exists),  I am curious about what happened for VScode to give me a sign, thank you.

**Comment by EngrPips** [comment, marked as answer]
Hello @lilsherryG , since your test pass actually did pass i would assume that your `vs code` is acting weird maybe close it and re-open it

**Reply by lilsherryG** [reply]
Thank you, done :)

**Reply by EngrPips** [reply]
Good , keep crushing it
